site/pages/react/send-user-operations.mdx

```diff
@@ -26,7 +26,7 @@ import {
 } from "@account-kit/react";
 
 export default function MyOpSenderComponent() {
-  const { client } = useSmartAccountClient({ type: "LightAccount" });
+  const { client } = useSmartAccountClient({});
```

Comment:
The above tool tip is no longer accurate since this is using MAv2 no right? @avarobinson 

```
In the below example, we use `LightAccount` as the underlying Smart Contract type. You can also use `MultiOwnerModularAccount` if you want to provide your users with an ERC-6900 compliant modular account,
```